backend/workflow_manager/endpoint/fs_connector_helper.py

```diff
@@ -0,0 +1,29 @@
+from pathlib import Path
+
+from workflow_manager.endpoint.constants import UnstractFSConnector
+
+
+class UnstractFsConnectorHelper:
+
+    @staticmethod
+    def get_fs_root_dir(fs_cls_name: str, root_path: str, input_dir: str) -> str:
+        """Returns the unstract fs connector root directory
+        For eg: "root/" path is appended for Gdrive connector
+                "/" path is appended for Dropbox connector
+                "/" path is preppended for all other connector
+
+        Args:
+            fs_cls_name (str): unstract fs class-name
+            root_path (str): root path of unstract fs connector
+            input_dir (str): connector directory provided by user
+
+        Returns:
+            str: unstract fs connector root directory
+        """
+        if fs_cls_name in UnstractFSConnector.GOOGLE_DRIVE_FS:
```

Comment:
Instead of using a separate constant - rely on the [pythonic way](https://stackoverflow.com/a/511060/12864260) to obtain the class name

Reply:
@chandrasekharan-zipstack I am using same thing only [here](https://github.com/Zipstack/unstract/pull/456/files#diff-f38c2a49b866a35b58c08a1ee78638f0fa2a54afd5ca902da609cc6be0f7a113R188) and [here](https://github.com/Zipstack/unstract/pull/456/files#diff-b3d98d93fd437a9e3d2246946276a92b5689359ab1f6464534735d6775d72f7dR163)

Just to make code cleaner, I have used the constant part.

Reply:
@kirtimanmishrazipstack if we do make a change in the class name, we'll have to update it here as well. In the name of readability we would have compromised on maintainability. If you initialize these constants from the class name itself - then I think it should be okay